openshift-ci/build-root/multistage/check-oc.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -x
+
+## Constants
+OC_BINARY="./oc"
```

Comment:
Wouldn't it be safer to use an absolute path?

Reply:
It does not make more sense if we add the absolute path instead. Actually the oc binary path is defined through the ```destination_dir``` of the job configuration https://github.com/openshift/release/pull/9431/files#diff-f04f6304bab9d5afb9c78addf7eb17a2R16, we are just using the relative path from the test container working dir ```/go/src/github.com/openshift/origin```.